### PR TITLE
local dev config for item-validation-service

### PR DESCRIPTION
**Pull request by malaffoon**

### CODE REVIEW

File: docs/LOCAL-DEV.md [hunk 1]
```diff
@@ -75,45 +77,23 @@ local:
...
       username: imrt_ingest
       password: *****
     imrt-search:
-      url: "jdbc:postgresql://localhost:5432/imrt"
       username: imrt_search
       password: *****
   rabbitmq:
     username: guest
     password: *****
-  services:
```
Comment (malaffoon) [inline docs/LOCAL-DEV.md]: Dave, let's discuss this Monday. It's nice having the complete list of ports in one location. But it's also having reasonable defaults in the application-local.yml files so this isn't necessary. Thoughts?

File: item-validation-service/src/main/resources/application-local.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    username: "${local.db.iat.username:iat}"
+    password: "${local.db.iat.password}"
+server:
+  port: ${local.services.item-validation-service.port:8087}
+ivs:
+  # TODO - what is a good local default? assume macos; where?
+  cptExecutablePath:
+itembank:
+  # TODO - is /ItemBankValidation different from /ItemBank?
+#  localBaseDir: \${HOME}/ItemBankValidation
```
Comment (malaffoon) [inline item-validation-service/src/main/resources/application-local.yml]: @bdrainer, any input on this question would be appreciated.

File: item-validation-service/src/main/resources/application.yml [hunk 3]
```diff
@@ -56,9 +56,6 @@ itembank:
     fullName: "Item Validation Service"
   enabled:
     database: true
-  services:
-    history:
-      url: "http://localhost:8086"
```
Comment (malaffoon) [inline item-validation-service/src/main/resources/application.yml]: Brett said i could  :)

File: item-validation-service/src/main/resources/application-local.yml (hunk 2 above)
Comment (bdrainer) [reply item-validation-service/src/main/resources/application-local.yml]: There is no difference.  It is just a folder location on disk.  It is where the app will pull down things from S3 (or way back it was GitLab).  IMS does it, IRS does, etc.  We pointed them to different places but they could all point to /ItemBank.  Most apps have a cleanup tasks that monitors this folder and deletes files older than some time period.

**Pull request merged**